tests/test_calibrateImage.py

```diff
@@ -111,8 +111,8 @@ def setUp(self):
         # will use those fluxes here, and hopefully can remove this.
         self.config.astrometry.magnitudeOutlierRejectionNSigma = 9.0
 
-        # Something about this test dataset prefers the older fluxRatio here.
-        self.config.star_catalog_calculation.plugins['base_ClassificationExtendedness'].fluxRatio = 0.925
+        # Something about this test dataset prefers a larger threshold here.
+        self.config.star_selector["science"].unresolved.maximum = 0.2
```

Comment:
Did you confirm that this was necessary, or is this just an extrapolation from how the previous value was different from the default?

Reply:
Sadly, it is necessary, and presumably for similar reasons.